Dockerfile

```diff
@@ -16,6 +16,9 @@ LABEL name="openshift-ansible" \
 
 USER root
 
+# Create a symlink to /opt/app-root/src so that files under /usr/share/ansible are accessible
+RUN mkdir -p /usr/share/ansible/ && ln -s /opt/app-root/src /usr/share/ansible/openshift-ansible
+
```

Comment:
I've added it only to simplify working with the image as the same paths are valid also with `Dockerfile` and not having two different trees for the two images.  I can drop this patch if you prefer.